framework/auth/cas.py

```diff
@@ -208,15 +213,23 @@ def get_logout_url(*args, **kwargs):
     """
     return get_client().get_logout_url(*args, **kwargs)
 
+
 def get_profile_url():
     """Convenience function for getting a profile URL for a user.
     """
     return get_client().get_profile_url()
 
+
```

Comment:
Thanks. I am for the style as well so I try to improve them on files (only related to my tickets) Changing too much may cause headaches of merge conflicts for everyone so I took this "little by little" approach. :)